.github/workflows/tf-deploy-basic.yaml

```diff
@@ -173,6 +172,11 @@ jobs:
         run: |
           echo "$SUMMARY" >> $GITHUB_STEP_SUMMARY
 
+      # Added this step because otherwise the UI shows a green checkmark
```

Comment:
Could this be added to the step above? 

I would remove the comment

Reply:
@swibrow 
1. We want to create anyway the summary, so we can't exit before that.
2. We don't want to exit at the summary step or this would be misleading because it would look like the summary step failed while it was the plan one that failed.

What do you think? I think like this is the cleanest.